tests/slow/DDMetricsExclude.txt

```diff
@@ -0,0 +1,35 @@
+testTitle=DDMetricsExclude_populate
```

Comment:
tests in the tests/slow directory will be run automatically by our simulation tests. This test is a performance, so to avoid running correctness tests on it, please move it under the main tests directory